notebooks/Diagnose-TCR-ECS-TCRE.ipynb

```diff
@@ -362,164 +434,137 @@
-       "climate_model model       scenario     variable                       todo         unit    region               \n",
-       "MAGICC6       unspecified abrupt-2xCO2 Surface Temperature            not_relevant K       World     2.191377   \n",
-       "                                       Radiative Forcing              not_relevant W / m^2 World     3.650000   \n",
-       "                                       Atmospheric Concentrations|CO2 not_relevant ppm     World   553.488000   \n",
+       "climate_model model       region scenario     todo         unit    variable                                     \n",
+       "MAGICC6       unspecified World  abrupt-2xCO2 not_relevant ppm     Atmospheric Concentrations|CO2  553.488000   \n",
+       "                                                           K       Surface Temperature               2.191377   \n",
+       "                                                           W / m^2 Radiative Forcing                 3.650000   \n",
        "\n",
        "time                                                                                               1897-01-01  \\\n",
-       "climate_model model       scenario     variable                       todo         unit    region               \n",
-       "MAGICC6       unspecified abrupt-2xCO2 Surface Temperature            not_relevant K       World     2.193655   \n",
-       "                                       Radiative Forcing              not_relevant W / m^2 World     3.650000   \n",
-       "                                       Atmospheric Concentrations|CO2 not_relevant ppm     World   553.488000   \n",
+       "climate_model model       region scenario     todo         unit    variable                                     \n",
+       "MAGICC6       unspecified World  abrupt-2xCO2 not_relevant ppm     Atmospheric Concentrations|CO2  553.488000   \n",
+       "                                                           K       Surface Temperature               2.193655   \n",
+       "                                                           W / m^2 Radiative Forcing                 3.650000   \n",
        "\n",
        "time                                                                                               1898-01-01  \\\n",
-       "climate_model model       scenario     variable                       todo         unit    region               \n",
-       "MAGICC6       unspecified abrupt-2xCO2 Surface Temperature            not_relevant K       World     2.195908   \n",
-       "                                       Radiative Forcing              not_relevant W / m^2 World     3.650000   \n",
-       "                                       Atmospheric Concentrations|CO2 not_relevant ppm     World   553.488000   \n",
+       "climate_model model       region scenario     todo         unit    variable                                     \n",
```

Comment:
Any idea why the order changed?

Reply:
I think because we sort the index by default in scmdata now